app/models/compute_resources/foreman/model/vmware.rb

```diff
@@ -206,6 +206,7 @@ def firmware_types
         "automatic" => N_("Automatic"),
         "bios" => N_("BIOS"),
         "efi" => N_("EFI"),
+        "uefi_sb" => N_("UEFI Secure Boot"),
       }
     end
```

Comment:
Consider moving this to the `ComputeResource` model to be used by both VMware and Libvirt.

Reply:
Sounds like a good idea, but I'd separate it from this PR just to keep it focused.

Reply:
After working on this feature for both Libvirt and VMware, I believe it makes sense to move this method and the `firmware_mapping` method to the ComputeResource model for shared use.